website/docs/r/apigatewayv2_integration.html.markdown

```diff
@@ -35,7 +35,7 @@ resource "aws_lambda_function" "example" {
 
 resource "aws_apigatewayv2_integration" "example" {
   api_id           = aws_apigatewayv2_api.example.id
-  integration_type = "AWS"
+  integration_type = "AWS_MOCK"
```

Comment:
Unless I'm misunderstanding, based on the [document you linked in the PR](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-api-integration-types.html), it seems that this should instead be `AWS_PROXY`. It doesn't appear that `AWS_MOCK` is a valid integration type, and `MOCK` appears to be used only for testing. If I've misunderstood, I'd be happy to take another look! 🙂 

```suggestion
  integration_type = "AWS_PROXY"
```